patches/server/0933-Fix-wrong-position-for-bed-spawn-loc.patch

```diff
@@ -0,0 +1,65 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Lulu13022002 <41980282+Lulu13022002@users.noreply.github.com>
+Date: Wed, 20 Jul 2022 14:16:25 +0200
+Subject: [PATCH] Fix wrong position for bed spawn loc
```

Comment:
There is already https://github.com/PaperMC/Paper/blob/master/patches/server/0896-Fix-OfflinePlayer-getBedSpawnLocation.patch.

Concerning this is now adding new API, I guess the existing patch should just be merged into this one or the other way around, crediting either you or MM as a co-author.